core/controllers/access_validators_test.py

```diff
@@ -854,7 +904,7 @@ def test_access_story_editor_page_without_logging_in(self) -> None:
             '%s/can_access_story_editor_page/%s' % (
                 ACCESS_VALIDATION_HANDLER_PREFIX, self.story_id
             ), expected_status_int=302
-        )
+    )
```

Comment:
Indent this by 4 to match the get_html_response line (since that's where the opening paren is)

Reply:
Done